psutil/TODO.aix

```diff
@@ -0,0 +1,30 @@
+AIX support is experimental and incomplete at this time.
+The following functions and methods are UNSUPPORTED on the AIX platform:
+
+    psutil.Process.memory_maps
+    psutil.Process.num_ctx_switches
+
+
+Process.io_counters read count is always 0
+
+
+TestSystemAPIs.test_pid_exists_2           there are pids in /proc that don't really exist
```

Comment:
You mean, say, a `/proc/54` directory exists but there's no such PID?

Reply:
Yes, AIX is weird like that.

Reply:
That's *really* unfortunate. How about doing this then?

```
def pids():
    """Returns a list of PIDs currently running on the system."""
    return [int(x) for x in os.listdir('/proc') if x.isdigit() \
            and  _psposix.pid_exists(int(x))]
```

Does this test pass?

Reply:
This fixes `test_pid_exists_2` but causes `test_pids` to break because `ps` still returns the processes that "don't really exist" and then we get this:

      File "/root/psutil/psutil/tests/test_posix.py", line 297, in test_pids
         self.fail("difference: " + str(difference))
    AssertionError: difference: [131076, 196614, 1048608, 1114146, 1179684, 1376298, 1638568]